app/src/main/java/com/shifty/app/model/Application.java

```diff
@@ -0,0 +1,76 @@
+package com.shifty.app.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "application")
+public class Application {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long applicationId;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "job_id", nullable = false)
+    @JsonIgnore
+    private Job aJob;
+
+    @OneToMany(mappedBy = "userApplication", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
It's better with: @OneToMany(mappedBy = _"application"_, cascade =
Because it will be a part of application class in the UserApplication class.
That can be changed later related with the name of variable.

Reply:
Correct. Here there's an mapped object of Application to a Set of UserApplication to create the relationship between the two entities.